week-2/implement/vat.js

```diff
@@ -8,3 +8,10 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function vat(num) {
```

Comment:
Can you think of a better name for this function - something with a verb will make this clearer.